.github/workflows/performance.yml

```
@@ -0,0 +1,211 @@
+          if (( $(echo "$PASS_RATE < 0.9" | bc -l) )); then
+            echo "Performance tests failed: Pass rate $PASS_RATE is below 90%"
+            exit 1
+          fi
+        else
+          echo "CI summary not found"
+          exit 1
+        fi
+        
+    - name: Comment PR with results
+      if: github.event_name == 'pull_request'
+      uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)